content/cpp/concepts/math-functions/terms/nearbyint/nearbyint.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'nearbyint()'
+Description: 'Round off the given argument(value) to the nearest integral value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `cmath` header defines `nearbyint()` function that returns the nearby integral value of a given argument. It takes a single argument as a floating point value and returns the round off value of the argument as an integer.
+
+## Syntax
+
+```py
+nearbyint(n)
+```
+
```

Comment:
Under this snippet, let's include 1-2 sentences that describe the `n` parameter including what data type(s) it can be and possible edge cases. For example, what would happen if `n` were equal to something like zero or pos/neg infinity or a float that ends in `.5` (rounds up or down)?